Header header
mask_rcnn/SemanticObject[] semantic_objects
sensor_msgs/Image rgb
sensor_msgs/Image depth
sensor_msgs/Image mask
sensor_msgs/Image flow


